src/modules/Expanders.jl

```diff
@@ -272,6 +272,102 @@ immutable DocsNodes
     nodes :: Vector{DocsNode}
 end
 
+"""
+Converts the function argument tuple `(name, type)` into a string.
+
+The tuple comes from the second return element of the `Base.arg_decl_parts(::Method)`
+and it seems they are always both `::String` (`::ASCIIString` in 0.4).
+It also appears that if the type is not declared for the method, `arg_decl_parts`
+returns an empty string.
+
+The returned string is `name::type` or just `name`, if the type is not declared.
+"""
+function _function_arg(decl)
+    n,t = decl
+    if length(t)==0
+        n
+    else
+        "$(n)::$(t)"
+    end
+end
+
+"""
+Returns a markdown paragraph (`Markdown.Paragraph`) with a list of methods.
+
+The return value will be wrapped in a `Nullable`, and a `null` is returned
+if the object in `ex` has no methods.
+"""
+function methodtable_md(mod::Module, ex, page, doc)
+    # TODO:
+    #   - fails on 0.4
+    #   - doesn't work with macros (problem with eval)
+    obj = try
+        mod.eval(ex)
```

Comment:
That's a good solution. So only display method/constructor lists when it's a `Function` or `DataType`? (Slightly unrelated, might be nice to add an export list to `Module` docs automatically, similar to this, at some point.)

> Hmm, I didn't know that was supported.

I'm not sure how common it's use is currently, but it can come in handy sometimes I think.